### PR TITLE
Ready for review: Meet our Founder Page

### PR DESCRIPTION
**Pull request by ridham-virufy**
Ready for review: Meet our Founder Page

### CODE REVIEW

**Comment by gevuong** [comment]
@ridham-virufy Does this PR cover mobile view as well? If so, it looks like the button extends beyond the web page.

Screenshot 2024-05-24 at 2 37 49 PM

**Review by a-wong-8 — COMMENTED** [review]
@gevuong Ridham said mobile is not done yet.

**Comment by gevuong** [comment]
@ridham-virufy it looks like the text is cut off in the last card:

Screenshot 2024-05-24 at 8 10 41 PM

**Comment by a-wong-8** [comment]
I think if you made the text smaller, it will match the figma more. 
Screenshot 2024-05-24 at 6 20 29 PM
Screenshot 2024-05-24 at 6 18 51 PM

**Comment by gevuong** [comment]
Looks like the text is cut off here as well
Screenshot 2024-05-24 at 8 15 48 PM

**Comment by ridham-virufy** [comment]
@a-wong-8 The text is fixed now the last card doesn't have text overflowing

**Review by gevuong — COMMENTED** [review]
I noticed there’s commented code in 5 different areas in your PR. Are they needed? If so, can you leave a comment explaining why? Otherwise, I recommend removing it.

The purpose of doing this is so that the code will be much cleaner and easier to understand, especially for the next person who looks at it.

**Review by a-wong-8 — COMMENTED** [review]
Screenshot 2024-05-28 at 10 13 17 AM
- amils img is missing alt text

**Comment by gevuong** [comment]
@ridham-dd as a best practice, please create a separate branch to store changes for the home page as the home page is a separate ticket and feature. This PR and branch should only contain changes in the meet our founders page (and for convenience, minor bug fixes to other pages).

I recommend reverting your home page changes in this branch as it's causing the build to fail. Lets first resolve the requested changes Anthony and I commented on for the meet our founders page so we can close this 2-week old PR. Otherwise, if changes to the home page was added to this PR, the code will become difficult to review and QA test (and debug if necessary).

**Comment by ridham-virufy** [comment]
Screenshot 2024-06-06 at 10 15 08 PM
Screenshot 2024-06-06 at 10 15 29 PM
Screenshot 2024-06-06 at 10 15 53 PM
Screenshot 2024-06-06 at 10 16 00 PM
Screenshot 2024-06-06 at 10 16 11 PM

### **QA comments closed, alt text for Amil's headshot added, re-facotred the code and removed excess comments, made the page responsive.**

**Review by a-wong-8 — COMMENTED** [review]
good job @ridham-virufy! But the text box is too high up on the mobile view which is covering up some of the advisors faces...

**Comment by ridham-virufy** [comment]
@a-wong-8 Yes I felt the same but a few advisors have a lot of text to be kept in that 50% opacity black filter and the text size was already xxs. That's why I made that filter responsive for desktop and mobile view, so that there is a good view in desktop mode. Let me know if you have any suggestion.

**Comment by a-wong-8** [comment]
You should be able to change the height of the text box for tablet and mobile view only.

File: src/components/molecules/Card/Card.js [hunk 1]
```diff
@@ -2,76 +2,52 @@ import Title from "~/components/atoms/Title/Title";
...
+        <div 
+          className={`flex flex-col items-center space-y-4 p-4 w-full ${remainder === 1 && index === CardsData.length - 1 ? 'lg:col-start-2 xl:col-start-2' : ''}`}
+          key={id}
+        >
+          <div className="relative w-full">
+            <div className="absolute top-4 lg:top-5 xl:top-7 z-10 w-full">
+              <Text
+                Style="subtitleCompact"
+                Text={subtitle}
+                TextClassProps="text-white font-semibold text-xs sm:text-xs lg:text-lg xl:text-xl sm:leading-tight sm:mb-1"
```
Comment (gevuong) [inline src/components/molecules/Card/Card.js:21]: similarly, `sm:text-xs` is redundant as `text-xs` affects `sm` breakpoints as well.

File: src/components/molecules/Card/Card.js [hunk 2]
```diff
@@ -2,76 +2,52 @@ import Title from "~/components/atoms/Title/Title";
...
-              Style="subtitle"
-              Text={subtitle}
-              TextClassProps="text-white"
-            />
-              </div>
-              <div>
+    <div className="grid grid-cols-2 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 gap-4 mb-4 text-center items-center justify-center mt-5 w-full lg:w-[80rem] xl:w-[80rem] mx-[10rem] lg:mx-[5rem]">
+      {CardsData?.map(({ route, name, text, subtitle, id }, index) => (
+        <div 
+          className={`flex flex-col items-center space-y-4 p-4 w-full ${remainder === 1 && index === CardsData.length - 1 ? 'lg:col-start-2 xl:col-start-2' : ''}`}
```
Comment (gevuong) [inline src/components/molecules/Card/Card.js:13]: `lg:col-start-2` affects `xl` breakpoints. `xl:cols-start-2` shouldn't be needed.

File: src/components/molecules/Card/Card.js [hunk 3]
```diff
@@ -2,76 +2,52 @@ import Title from "~/components/atoms/Title/Title";
...
-              style={{
-                position: 'absolute',
-                top: 300,
-                left: 0,
-                right: 0,
-                bottom: 0,
-                paddingBottom: 0 
-              }}
+            <div
+              className="absolute inset-x-0 bottom-0 bg-black bg-opacity-50 p-1 top-[54%] sm:top-[54%] lg:top-[70%] xl:top-[70%]"
```
Comment (gevuong) [inline src/components/molecules/Card/Card.js:32]: `lg:top-[70%]` should also apply to `xl` breakpoint. `xl:top-[70%]` becomes redundant.

**Comment by gevuong** [comment]
@ridham-dd Please remove `xl:` breakpoint classes if the same class is applied to an `lg:` breakpoint in the same HTML element. For example, if `lg:hidden` and `xl:hidden` are applied to the same element, `xl:hidden` becomes redundant and can be removed. I saw multiple instances of this occurring but decided to leave a single comment to cover all the cases. This also applies to other breakpoints as well, for example: `text-center sm:text-center` can just be `text-center`.

This explains why: https://tailwindcss.com/docs/responsive-design#working-mobile-first

**Comment by a-wong-8** [comment]
I was playing with it and looks like you can change the top to 76% for smaller screens to fix the text box issue.  
Screenshot 2024-06-07 at 10 26 58 AM

**Comment by ridham-virufy** [comment]
@a-wong-8 
image
For me, the text goes out of the screen when the top is 76% for small screens

**Comment by a-wong-8** [comment]
@ridham-virufy play with it and the best way

**Comment by ridham-virufy** [comment]
@a-wong-8 I made a new commit with the top fixed in mobile view, Here are the images. 
Screenshot 2024-06-07 at 12 54 33 PM
Screenshot 2024-06-07 at 12 54 46 PM


@gevuong As suggested, I have removed the extra breakpoints that were present which was making the code redundant. Let me know what you both think.

**Comment by gevuong** [comment]
@ridham-dd There are still redundant classes. Please review your code changes.

**Comment by ridham-virufy** [comment]
@gevuong I am pretty sure that all the redundant code is removed. I missed it out on the first glimpse, let me know what you think. It should be good for merging now.

**Review by a-wong-8 — APPROVED** [review]
Ridham is the man

**Pull request merged**